app/services/usersDataBase.js

```diff
@@ -0,0 +1,7 @@
+const { user: User } = require('../models');
+
+exports.userAlreadyExists = email =>
+  User.findAndCountAll({
```

Comment:
you dont need and find and count all, just a find would be fine!

Reply:
ok!